src/app/data-explorer/specification-submission/services/submission.broadcast.service.ts

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2022-2023 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+import { Injectable } from '@angular/core';
+import { BroadcastService } from 'src/app/core/broadcast.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SubmissionBroadcastService {
```

Comment:
Just wondering why you didn't include this in the existing broadcast service?

Reply:
I thought the broadcast service was a more general service to be utilised. But I now see that there is stuff in their that is specific to users, so I have got rid of the SubmissionBroadcastService and moved the new code into the broadcastService.